Code/GraphMol/FileParsers/FileParserUtils.h

```diff
@@ -32,26 +33,133 @@ T stripSpacesAndCast(const std::string &input, bool acceptSpaces = false) {
     return boost::lexical_cast<T>(trimmed);
   }
 }
-RDKIT_FILEPARSERS_EXPORT int toInt(const std::string &input, bool acceptSpaces = false);
-RDKIT_FILEPARSERS_EXPORT double toDouble(const std::string &input, bool acceptSpaces = true);
+RDKIT_FILEPARSERS_EXPORT int toInt(const std::string &input,
+                                   bool acceptSpaces = false);
+RDKIT_FILEPARSERS_EXPORT double toDouble(const std::string &input,
+                                         bool acceptSpaces = true);
 
 // reads a line from an MDL v3K CTAB
-RDKIT_FILEPARSERS_EXPORT std::string getV3000Line(std::istream *inStream, unsigned int &line);
+RDKIT_FILEPARSERS_EXPORT std::string getV3000Line(std::istream *inStream,
+                                                  unsigned int &line);
 
 // nAtoms and nBonds are ignored on input, set on output
-RDKIT_FILEPARSERS_EXPORT bool ParseV3000CTAB(std::istream *inStream, unsigned int &line, RWMol *mol,
-                    Conformer *&conf, bool &chiralityPossible,
-                    unsigned int &nAtoms, unsigned int &nBonds,
-                    bool strictParsing = true, bool expectMEND = true);
+RDKIT_FILEPARSERS_EXPORT bool ParseV3000CTAB(
+    std::istream *inStream, unsigned int &line, RWMol *mol, Conformer *&conf,
+    bool &chiralityPossible, unsigned int &nAtoms, unsigned int &nBonds,
+    bool strictParsing = true, bool expectMEND = true);
 
 // nAtoms and nBonds are used
-RDKIT_FILEPARSERS_EXPORT bool ParseV2000CTAB(std::istream *inStream, unsigned int &line, RWMol *mol,
-                    Conformer *&conf, bool &chiralityPossible,
-                    unsigned int &nAtoms, unsigned int &nBonds,
-                    bool strictParsing = true);
+RDKIT_FILEPARSERS_EXPORT bool ParseV2000CTAB(
+    std::istream *inStream, unsigned int &line, RWMol *mol, Conformer *&conf,
+    bool &chiralityPossible, unsigned int &nAtoms, unsigned int &nBonds,
+    bool strictParsing = true);
 
 RDKIT_FILEPARSERS_EXPORT Atom *replaceAtomWithQueryAtom(RWMol *mol, Atom *atom);
+
+template <typename T>
+void applyMolListPropsToAtoms(ROMol &mol, const std::string &prefix,
+                              const std::string &missingValueMarker = "n/a") {
+  for (auto pn : mol.getPropList()) {
+    if (pn.find(prefix) == 0 && pn.size() > prefix.size()) {
+      std::string strVect = mol.getProp<std::string>(pn);
```

Comment:
Does std::string always lexically cast here?  What if, for some reason, the prop "pn" isn't a string?  What if it's an any object or something.

I might wrap this in a try catch...  perhaps a regression test is in order.